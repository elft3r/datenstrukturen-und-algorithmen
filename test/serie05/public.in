7
2 3 1
4 -6 -3 6 -6
3 3 -4 5
5 -5 2 9 -1 1
8 8 15 9 9 15 6 -1 -11
10 -12 -5 -10 -12 13 -19 -4 17 -13 -9
50 41 16 68 -17 29 -8 29 -100 -86 -50 -12 33 11 88 74 90 -55 67 -42 71 -97 -94 -11 -70 -27 -44 29 -3 -69 -8 -96 -99 -93 -41 -18 -27 57 -51 -28 -55 -74 -74 9 16 47 2 54 -12 52 20
