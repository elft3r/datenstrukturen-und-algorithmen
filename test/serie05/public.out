1 0 0 3 1 0 
-6 0 0 6 0 0 -3 0 1 -6 1 2 
-4 0 0 5 0 0 3 1 1 
1 0 0 -1 0 1 9 0 0 2 2 1 -5 0 3 
-11 0 0 -1 1 0 6 2 0 9 0 0 15 0 0 9 1 1 15 2 0 8 3 3 
-13 0 0 -19 0 1 -12 2 0 -9 0 0 -10 0 1 -4 0 0 17 0 0 13 1 1 -5 2 2 -12 3 3 
-99 0 0 -96 0 0 -93 0 0 -94 1 1 -97 1 2 -74 0 0 -74 1 0 -55 0 0 -69 0 1 -70 2 2 -51 0 0 -55 3 1 -44 0 0 -28 0 0 -27 1 0 -41 0 2 -18 0 0 -27 3 1 -42 1 4 -50 4 5 -86 3 6 -100 0 7 -12 0 0 -8 0 0 -11 0 1 -12 1 2 2 0 0 9 1 0 -3 0 2 16 0 0 11 3 1 -8 3 4 -17 8 5 20 0 0 29 1 0 29 2 0 33 0 0 29 3 1 16 9 4 52 0 0 54 1 0 47 0 2 57 3 0 67 4 0 71 0 0 74 1 0 90 0 0 88 2 1 68 5 3 41 10 6 
